src/GafferCycles/CyclesOptions.cpp

```diff
@@ -122,6 +123,17 @@ CyclesOptions::CyclesOptions( const std::string &name )
 	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_denoise_pass_normal", new IECore::BoolData( true ), false, "useDenoisePassNormal" ) );
 	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:denoiser_prefilter", new IECore::StringData( "accurate" ), false, "denoiserPrefilter" ) );
 
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_guiding", new IECore::BoolData( false ), false, "useGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:deterministic_guiding", new IECore::BoolData( true ), false, "deterministicGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_surface_guiding", new IECore::BoolData( true ), false, "useSurfaceGuiding" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:surface_guiding_probability", new IECore::FloatData( 0.5f ), false, "surfaceGuidingProbability" ) );
+	options->addChild( new Gaffer::NameValuePlug( "cycles:integrator:use_volume_guiding", new IECore::FloatData( true ), false, "useVolumeGuiding" ) );
```

Comment:
Seems like this option should be `BoolData` rather than `FloatData`?

https://github.com/blender/cycles/blob/3eaf89c3c0b600eabc3d2fd8361fa17abc8f8992/src/scene/integrator.cpp#L72

Reply:
Ahh damn typos, fixed!